From 3184b8d27c919ebe2193d6b7ee05bd86e202ea65 Mon Sep 17 00:00:00 2001
From: Stefan Wahren <wahrenst@gmx.net>
Date: Sat, 21 Sep 2019 15:29:33 +0200
Subject: [PATCH 548/587] ARM: dts: bcm2708-rpi: Define the downstream MMC
 interfaces at one place

In order to keep downstream definition of the MMC interfaces separate
and independent from upstream changes, add them to bcm2708-rpi.dtsi.

Signed-off-by: Stefan Wahren <wahrenst@gmx.net>
---
 arch/arm/boot/dts/bcm2708-rpi.dtsi | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm/boot/dts/bcm2708-rpi.dtsi b/arch/arm/boot/dts/bcm2708-rpi.dtsi
index a2e0166ce3c6..ca072d2c6288 100644
--- a/arch/arm/boot/dts/bcm2708-rpi.dtsi
+++ b/arch/arm/boot/dts/bcm2708-rpi.dtsi
@@ -125,13 +125,23 @@
 	firmware = <&firmware>;
 };
 
+&sdhci {
+	pinctrl-names = "default";
+	pinctrl-0 = <&emmc_gpio48>;
+	bus-width = <4>;
+};
+
 sdhost_pins: &sdhost_gpio48 {
 	/* Add alias */
 };
 
 &sdhost {
+	pinctrl-names = "default";
+	pinctrl-0 = <&sdhost_gpio48>;
+	bus-width = <4>;
 	brcm,overclock-50 = <0>;
 	brcm,pio-limit = <1>;
+	status = "okay";
 };
 
 &cpu_thermal {
-- 
2.17.1

